package/src/Luna/Package.hs

```diff
@@ -207,26 +209,35 @@ listDependencies pkgSrc = Exception.rethrowFromIO @Path.PathException $ do
             pure $ fmap (convert &&& (lunaModulesPath FilePath.</>)) directDeps
                   <> concat indirectDeps
 
+includedLibs :: MonadIO m => m [(Name.Name, FilePath.FilePath)]
+includedLibs = do
+    lunaroot     <- liftIO $ Directory.canonicalizePath
+        =<< Environment.getEnv Name.lunaRootEnv
+    projectNames <- liftIO $ do
+        contents <- Directory.listDirectory lunaroot
+        dirs     <- filterM
+            (\a -> Directory.doesDirectoryExist $ lunaroot FilePath.</> a) contents
+        let projects = filter
+                (\a -> (isUpper <$> head a) == Just True) dirs
+        return projects
+    for projectNames $ \projectName ->
+        pure (convert projectName, lunaroot <> "/" <> projectName <> "/")
```

Comment:
This is non-portable to windows due to the hardcoded `"/"`. You need to use either `</>` or `pathSeparator` from `System.FilePath`.

Reply:
It was tested on Windows and worked fine

Reply:
Interesting! I'd still prefer use of `pathSeparator`, but if it works that's purely a cosmetic nitpick. Approved!

Reply:
Windows itself (as in NT kernel) accepts both `\` and `/`, you can even freely mix them together. However many applications (including cmd and explorer) assume backslashes as separators and don't fully support forward slashes. It is strongly recommended to use `\` as path separator for consistency and so path eg. can be easily pasted into explorer.